keyboards/keebio/levinson/keymaps/issmirnov/push-left.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
These scripts should really go as well. Use the CLI instead, you can run it from anywhere too.

Reply:
Done.